dask/dataframe/tests/test_dataframe.py

```diff
@@ -306,19 +306,76 @@ def test_describe(method, test_values):
     assert_eq(df.describe(), ddf.describe(split_every=2, percentiles_method=method))
 
 
-@pytest.mark.parametrize('method', ['tdigest', 'dask'])
```

Comment:
This parametrization probably shouldn't have been dropped from `test_describe_empty`.